dev/com.ibm.ws.collector.manager/test/com/ibm/ws/collector/manager/buffer/RingBufferTest.java

```diff
@@ -377,7 +374,7 @@ public void assertBlocked(String message, Thread thread) {
     public void assertNotBlocked(String message, Thread thread) {
 
         long threadWaitTimeOutInMilliSecs = 10000;
-        long waitTimeInMilliSecs = 1000;
+        long waitTimeInMilliSecs = 1500;
```

Comment:
There is a retry in the following while loop until 10 secs is reached.  Not sure changing the wait from 1 sec to 1.5 secs would help.

Reply:
Ah, yes you're right.
Will have to wait for the next occurrence to see what debug output was since last occurrence was may 10.